### ISSUE TITLE
Adjust frontend button styles

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2359]
This green button has insufficient contrast:
![image](https://user-images.githubusercontent.com/255909/65057383-64d2cf80-d927-11e9-9d8e-1d366f9e7e39.png)

We can leave it as-is in the backend, but in the frontend when it appears over white, we need to change it to this:
![image](https://user-images.githubusercontent.com/255909/65057414-71572800-d927-11e9-91c8-7c453ee5ef9d.png)
